public/app/features/explore/SecondaryActions.tsx

```diff
@@ -34,32 +35,32 @@ export function SecondaryActions(props: Props) {
         {!props.addQueryRowButtonHidden && (
           <ToolbarButton
             variant="canvas"
-            aria-label="Add query"
+            aria-label={t('explore.secondary-actions.queryadd-button-label', 'Add query')}
             onClick={props.onClickAddQueryRowButton}
             disabled={props.addQueryRowButtonDisabled}
             icon="plus"
           >
-            Add query
+            <Trans i18nKey="explore.secondary-actions.query-add-button">Add query</Trans>
           </ToolbarButton>
         )}
         {!props.richHistoryRowButtonHidden && (
           <ToolbarButton
             variant={props.richHistoryButtonActive ? 'active' : 'canvas'}
-            aria-label="Query history"
+            aria-label={t('explore.secondary-actions.queryhistory-button-label', 'Query history')}
             onClick={props.onClickRichHistoryButton}
             data-testid={Components.QueryTab.queryHistoryButton}
             icon="history"
           >
-            Query history
+            <Trans i18nKey="explore.secondary-actions.queryhistory-button">Query history</Trans>
           </ToolbarButton>
         )}
         <ToolbarButton
           variant={props.queryInspectorButtonActive ? 'active' : 'canvas'}
-          aria-label="Query inspector"
+          aria-label={t('explore.secondary-actions.queryinspector-button-label', 'Query inspector')}
```

Comment:
```suggestion
          aria-label={t('explore.secondary-actions.query-inspector-button-aria-label', 'Query inspector')}
```

Reply:
@ifrost Thank you for your feedback.

Reply:
@ifrost I have implemented your suggestions for improvements. Would you review my PR? I'm open to making any required changes further for this issue. Thank you.

Reply:
Thanks @siddhikhapare! Could also solve conflict? In the case of `.json` files the easiest way might be to take everything main and re-create all translations you have added again.